### ISSUE TITLE
Fix formatting for top bar of Map to render correctly on all screens

### ISSUE THREAD

**Issue by beegeiger** [beegeiger/Safework-Project#18]
The bar at the top of the Map page which includes the year sliders renders correctly on chrome on a computer, but doesn't always depending on the browser or device. Fix the CSS so that it looks good in every context.

**Comment by Satyabrat35** [comment]
You want to say that the year slider doesn't get render properly in every browser or device? 
Am i correct? or is it something else?

**Comment by beegeiger** [comment]
Exactly, particularly on mobile, the buttons in the bar can appear way too big or small and can even awkwardly stack so they don't fit in the top bar.
